src/hps_align/detdump/_global.py

```diff
@@ -139,19 +142,35 @@ def global_coord(
             v = [float(line_elements[i][:-1]) for i in range(10, 13)]
             w = [float(line_elements[i][:-1]) for i in range(14, 17)]
             sensor_map[name] = dict(
-                position=position,
+                hps_position=position,
                 u=u, v=v, w=w
             )
+        if svtFrameCount:
+            svtFrameLine += 1
+        if svtFrameLine == 14:
+            svtFrameLine = 0
+            svtFrameCount = False
+            line_elements = line.strip().split()
+            position = [float(line_elements[i][:-1]) for i in range(1, 4)]
+            sensor_map[name] = dict(
+                hps_position=sensor_map[name]['hps_position'],
+                svt_position=position,
+                u=sensor_map[name]['u'],
+                v=sensor_map[name]['v'],
+                w=sensor_map[name]['w']
+            )
```

Comment:
You can assign keys in place

```suggestion
            sensor_map[name]['svt_position'] = position
```